docs/Developers/Writing_Linter_Bears.rst

```diff
@@ -398,6 +398,8 @@ Now we need to add some more precious information to our bear. This helps
 by giving more information about each bear and also helps some functions
 gather information by using these values. Our bear now looks like:
 
+.. include:: /home/imrekcah/coala/docs/Developers/language_naming_format.rst
```

Comment:
Netlify Deploy is showing that this path doesn't exist. Are you sure this path is correct?

```3:53:15 PM: /opt/build/repo/docs/Developers/Writing_Linter_Bears.rst:401:Problems with "include" directive path:```
```3:53:15 PM: InputError: [Errno 2] No such file or directory: 'home/imrekcah/coala/docs/Developers/language_naming_format.rst'.```

Reply:
@frextrite @shashank-b   Hello brother I tried a lot of paths can you please help me with the path